src/icons/slow_off.svg

```diff
@@ -0,0 +1,8 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 36.9 17.8" xml:space="preserve">
+  <style>
+    .st0{fill:#375273}
+  </style>
```

Comment:
@sledgehammer999:  You can inline the style in this file, too. 

Find-and-replace `class="st0"` with `fill="#375273"`, then delete the `<style>` section.